examples/with_winit/src/lib.rs

```diff
@@ -138,6 +138,9 @@ fn run(
                                         * transform;
                                 }
                             }
+                            Some(VirtualKeyCode::Space) => {
```

Comment:
I wonder if <kbd>R</kbd> would be a better choice here. The use of <kbd>Space</kbd> is a remnant from #260 , and this wasn't discussed in there.

I don't have a strong preference any way, but I'd like to leave this open for a bit to let anyone else comment.

(If this is changed, the README would also need to be updated)